lib/components/helpers/measureweather.js

```diff
@@ -0,0 +1,17 @@
+function measureWeather(temp) {
+  let average = (temp.high+temp.low)/2
+  if (average > 75) {
```

Comment:
Have you ever tried using a switch/case function for these?  I don't think it's necessarily better but something to consider as another tool in the future!